lib/cn/cn_tree.c

```diff
@@ -1745,34 +1805,36 @@ cn_comp_commit(struct cn_compaction_work *w)
         cn_comp_update_kvcompact(w, kvsets[0]);
         break;
 
-    case CN_ACTION_SPILL:
-        cn_comp_update_spill(w, kvsets);
-        break;
-
     case CN_ACTION_SPLIT:
         cn_comp_update_split(w, kvsets, split_nodev);
         break;
+
+    case CN_ACTION_SPILL:
+        assert(0);
+        break;
```

Comment:
There were in enumeration order :-)

Reply:
Of course :)
Not sure why I changed the ordering here. I'll restore it.